src/bootstrap/src/core/config/tests.rs

```diff
@@ -66,6 +66,9 @@ fn detect_src_and_out() {
 
         // test if build-dir was manually given in config.toml
         if let Some(custom_build_dir) = build_dir {
+            // FIXME(jieyouxu): this assertion fails on native Windows because if the rust checkout
+            // and build directory is on `E:\\` not `C:\\`, then it fails with e.g. `E:\\tmp !=
+            // C:\\tmp`.
             assert_eq!(&cfg.out, Path::new(custom_build_dir));
```

Comment:
This fails locally for me, thus I opened #129188. I should change this to `FIXME(#129188)` later.